README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
```

Comment:
We don't know yet if it will be lightweight 🤪

Reply:
I asked for some cool tech adjectives but no one replayed so I just called it lightweight. I do think that having lightweight as a gool is overall good for the project. We need some way to sound trendy and cool 😄

Reply:
Then what should I write here? "Unexciting project" or perhaps skip this part entirely?

Reply:
maybe 
```suggestion
A plugin for managing large minecraft servers
```